docs/src/how-to-use.md

```diff
@@ -0,0 +1,11 @@
+# [How to Use](@id how-to-use)
+
+## [Install](@id install)
+
+* Clone the repository from [TulipaBulb.jl](https://github.com/TNO-Tulipa/TulipaBulb.jl) into your local machine
+* Open the project in your favorite IDE (e.g., [Visual Studio Code](https://code.visualstudio.com/))
+* Start a Julia REPL
+* `]`: Enter package mode
+* `activate .` : Activate here (root of project)
+* `instantiate` : Instantiate any packages you need according to the Project.toml
+* `test`: Run tests - they should pass before you create a pull request
```

Comment:
please indent this part

Reply:
Thanks! The new version has the indentation :)